users/tests/test_unit_validators.py

```diff
@@ -44,3 +48,41 @@ def _assert_date_of_birth_is_valid(self, date_of_birth):
             self.user_age_validator(date_of_birth)
         except Exception as e:  # pylint:disable=broad-except
             self.fail(f"Unexpected exception {str(e)} has occurred!")
+
+
+class TestUserEmailValidation(TestCase):
+    def setUp(self) -> None:
+        self.user_email_validation = UserEmailValidation()
+
+    @parameterized.expand(
+        [
+            ("testusercodepoets.it", ValidationErrorText.VALIDATION_ERROR_EMAIL_AT_SIGN_MESSAGE),
+            ("@codepoets.it", ValidationErrorText.VALIDATION_ERROR_EMAIL_MALFORMED_FIRST_PART),
+            ("testuser@invaliddomain.com", ValidationErrorText.VALIDATION_ERROR_EMAIL_MESSAGE_DOMAIN),
+        ]
+    )
+    def test_validator_should_throw_exception_with_invalid_email(self, email, exception_message):
+        with self.assertRaises(ValidationError) as exception:
+            self.user_email_validation(email)
+        self.assertEqual(exception_message, exception.exception.message)
+
+    @parameterized.expand(
+        [
+            ("valid@codepoets.it", ["codepoets.it"]),
+            ("valid@otherdomain.it", ["otherdomain.it"]),
```

Comment:
This test case check exactly the same thing as one with domain `codepoets.it`. You should check here what happen (until we don't have system checks) what will happened if someone forgot to add any valid domain to this list.

Reply:
Oh, I see same test what I mentioned in comment above is at the bottom of this file. So please just delete this one with `otherdomain.it`

Reply:
Yes, I deleted this and one from test below.